### PR TITLE
issue #1412 Fixed the bugs in footer

### PR DESCRIPTION
**Pull request by swaraj-das**
## Description

This pull request aims to improve the alignment of the section headings in the footer. The adjustments ensure that the headings are properly aligned for a cleaner and more professional appearance. This change enhances the overall visual consistency and readability of the footer section.

## Issue

- Closed issue #1412

## Type of PR

- [X] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [X] Other (specify): Improved footer responsiveness across all pages.

## Screenshots / videos (if applicable)
**Before :**
- The section headings in the footer were misaligned, leading to an inconsistent and cluttered appearance.
- The footer lacked responsiveness, causing layout issues on various screen sizes.
  
foodie_before
mobile_view_before
  
**After :**
- The section headings in the footer are now properly aligned, providing a clean and professional look.
- The footer has been made fully responsive, ensuring it displays correctly across all devices and screen sizes.
  
foodie_after
mobile_view_after
  
## Checklist

- [X] I have gone through the [contributing guide](https://github.com/khushi-joshi-05/Food-ordering-website/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### Additional Context

The alignment and responsiveness improvements were achieved by adjusting the CSS properties for the footer section. These changes ensure that the footer's layout remains consistent and visually appealing on different devices and screen sizes. This enhancement not only improves the aesthetics but also enhances the user experience by providing a more organized and accessible footer.

### CODE REVIEW

File: Css-files/footer.css [hunk 1]
```diff
@@ -31,6 +32,8 @@ footer {
   background-color: rgba(210, 167, 167, 0.5);
   color: white;
   border: none;
+  width: 100%; /* Adjust the width as needed */
+  width: 500px; /* Add a max-width for larger screens */
```
Comment (swaraj-das) [inline Css-files/footer.css:36]: Added width of foot-panel1 in the css file...removing from html inline styling.

File: Css-files/footer.css [hunk 2]
```diff
@@ -53,6 +56,15 @@ footer {
   border: none;
 }
 
+/* Center h5 tags in desktop view */
```
Comment (swaraj-das) [inline Css-files/footer.css:59]: Made the H5 tags in the center of the columns.

File: Css-files/footer.css [hunk 3]
```diff
@@ -133,15 +150,45 @@ footer {
...
+.media-handles h4:hover {
+  color: #d24b29;
 }
+
 .row h5:hover,
 .row h3:hover {
-  color:#d24b29;
+  color: #d24b29;
+}
+
```
Comment (swaraj-das) [inline Css-files/footer.css:162]: Created footer responsive for different screen size.

File: index.html [hunk 4]
```diff
@@ -895,85 +895,45 @@ <h2 class="text-center">Download This Amazing App Now!</h2>
         <div class="foot-panel1 w-100">
             <div class="container-fluid py-5 px-0">
                 <div class="row justify-content-evenly gy-5">
-                    <div class="col-6 col-sm-3 col-md-2 col-xl-2">
-                        <h5 class="mb-3" style="margin-left: 70px;">See Also</h5>
+                    <!-- See Also Section -->
```
Comment (swaraj-das) [inline index.html:898]: Adjusted the column and row numbers and formatted the code with prettier extension.

**Review by swaraj-das — COMMENTED** [review]
Check the code and comments to review changes.

**Comment by sunny0625** [comment]
solve conflicts.

**Pull request closed without merging**